app/javascript/styles/_global.scss

```diff
@@ -791,3 +791,8 @@ figcaption.ck-editor__editable {
 .min-width-100 {
   min-width: 100%;
 }
+
+.file-upload-download-container {
+  display: flex;
+  align-items: center;
+}
```

Comment:
this seems like a set of styles that will be useful for a lot of form layouts.  We already have a class 'flexbox' for `display: flex`.  Maybe change this class name to `.flexbox-align-center` or something similar so we can reuse it elsewhere